MekHQ/src/mekhq/campaign/personnel/Person.java

```diff
@@ -2547,6 +2552,7 @@ private void parseRoleForDesignation(int role, StringBuilder sb) {
             case T_VTOL_PILOT:
             case T_VEE_GUNNER:
             case T_CONV_PILOT:
+            case T_VEHICLE_CREW:
                 sb.append("Lambda");
```

Comment:
I gotta ask, any idea what the deal is with this?

Reply:
ComStar designations I thought.